src/app/app-routing.module.ts

```diff
@@ -101,4 +105,4 @@ export const routes: Routes = [
   providers: [],
   exports: [RouterModule]
 })
-export class AppRoutingModule {}
+export class AppRoutingModule { }
```

Comment:
We should review the configuration of the project to ensure that lint fixes are applied only based on eslint configuration. The solution should also be portable so that all contributors fix lint issues the same way. I'll create a ticket for this. For now, let's stick to the convention in place.

Reply:
I am using the TypeScript and JS auto-formatter from VSCode.  I'll disable for now if it's breaking lint.

Reply:
Which linter are you using?  I was not getting any errors, hence why I did not bother with the spaces.  I will update my settings to match everyone else's.